packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -159,4 +171,12 @@ class TapGestureRecognizer extends PrimaryPointerGestureRecognizer {
 
   @override
   String get debugDescription => 'tap';
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder description) {
+    super.debugFillProperties(description);
+    description.add(new FlagProperty('wonArena', value: _wonArenaForPrimaryPointer, ifTrue: 'wonArena'));
+    description.add(new DiagnosticsProperty<Offset>('finalPosition', _finalPosition, defaultValue: null));
+    description.add(new FlagProperty('sentTapDown', value: _wonArenaForPrimaryPointer, ifTrue: 'sentTapDown'));
```

Comment:
the value here is wrong... i guess that means you need to add a test to verify the debugFillProperties/toString output. :-)

Reply:
similarly here, let's have ifTrue be `sent tap down`.